by-language/csharp-npgsql/README.rst

```diff
@@ -21,10 +21,8 @@ data provider for PostgreSQL, `crate-npgsql`_. CrateDB versions 4.2 and later
 work with the vanilla `Npgsql - .NET Access to PostgreSQL`_ driver without the
 need for a plugin.
 
-Please note that Npgsql 5 is not supported starting with CrateDB 4.8.4, you
-will need Npgsql 6 or newer.
-
-.NET 7, 8, and 9 are supported, .NET 3.1, 4.6, 5.0, and 6.0 may still work.
+The example program in this folder is validated on .NET 8 and 9,
```

Comment:
Is it fair to say that the example will only work when using Npgsql 8.x and 9.x since `EnableDynamicJson()` was introduced on 8.0?

Reply:
Yes. Do you think it needs an update?

Reply:
Well, it is not incorrect what it says. And since Npgsl 7.x and earlier are not supported, it is unlikely that a user will try with anything by 8.x or 9.x

Reply:
Let's leave it as it is